source/_integrations/mqtt.markdown

```diff
@@ -147,7 +147,7 @@ Add the MQTT integration, then provide your broker's hostname (or IP address) an
 
 1. Go to **{% my integrations title="Settings > Devices & services" %}**.
 2. Select the MQTT integration.
-3. Select **Configure**, then **Re-configure MQTT**.
+3. Select **Add**, then **Re-configure MQTT**.
```

Comment:
I think this is also related to an already existing config entry, those related to the options flow?